applications/osb-portal/src/components/MainDrawer/index.tsx

```diff
@@ -0,0 +1,264 @@
+import * as React from "react";
+//theme
+import {makeStyles, useTheme} from '@mui/styles';
+
+import {
+    bgRegular,
+    bgDarkest,
+    secondaryColor,
+    drawerText,
+    linkColor
+} from "../../theme";
+
+//hooks
+import useMediaQuery from '@mui/material/useMediaQuery';
+
+//components
+import {
+    ListItem,
+    ListItemIcon,
+    List,
+    ListItemText,
+    Toolbar,
+    Drawer,
+    Typography,
+    IconButton,
+    ListSubheader,
+    Box,
+    Button
+} from "@mui/material";
```

Comment:
This is fine but better to import every single component separately for performance as with this every single piece of material is included